src/sqlfluff/dialects/dialect_postgres.py

```diff
@@ -1335,6 +1335,106 @@ class AlterTableActionSegment(BaseSegment):
     )
 
 
+@postgres_dialect.segment()
+class CreateMaterializedViewStatementSegment(BaseSegment):
+    """A `CREATE MATERIALIZED VIEW` statement.
+
+    As specified in https://www.postgresql.org/docs/13/sql-creatematerializedview.html
+    """
+
+    type = "create_materialized_view_statement"
+
+    match_grammar = StartsWith(Sequence("CREATE", "MATERIALIZED", "VIEW"))
+
+    parse_grammar = Sequence(
+        "CREATE",
+        "MATERIALIZED",
+        "VIEW",
+        Ref("IfNotExistsGrammar", optional=True),
+        Ref("TableReferenceSegment"),
+        Ref("BracketedColumnReferenceListGrammar", optional=True),
+        AnyNumberOf(
+            Sequence("USING", Ref("ParameterNameSegment"), optional=True),
+            Sequence("TABLESPACE", Ref("ParameterNameSegment"), optional=True),
+            Sequence(
+                "WITH",
+                Bracketed(
+                    Delimited(
+                        Sequence(
+                            Ref("ParameterNameSegment"),
+                            Ref("EqualsSegment"),
+                            Ref("LiteralGrammar"),
+                            optional=True,
+                        ),
```

Comment:
```suggestion
                        Sequence(
                            Ref("ParameterNameSegment"),
                            Sequence(
                                Ref("EqualsSegment"),
                                Ref("LiteralGrammar"),
                                optional=True,
                            ),
                        ),
```
looking in the docs the equals value part is optional: `storage_parameter [= value]`